osgar/lib/test_quaternion.py

```diff
@@ -42,5 +43,9 @@ def test_from_axis_angle(self):
         self.assertAlmostEqual(expected[2], actual[2], places=6)
         self.assertAlmostEqual(expected[3], actual[3], places=6)
 
+    def test_asin_domain(self):
+        q = (0.02089575225593203, -0.6597655110187111, 0.381406681115162, -0.7726979260818011)
+        angles = quaternion.euler_zyx(q)  # should not raise ValueError: math domain error
+        self.assertTrue(almost_equal(angles, [-1.5436762, 1.0194396, -1.0505728]))
```

Comment:
What's wrong with [assertAlmostEqual](https://docs.python.org/3.6/library/unittest.html#unittest.TestCase.assertAlmostEqual)?

Reply:
it does not compare arrays of floats. An alternative is `numpy.testing`.

Reply:
The values of the angles don't match for what https://www.andre-gaschler.com/rotationconverter/ returns for the above quaternion: [ x: 0.9934288, y: 1.0734897, z: -1.5410159 ] That seems strange.

Reply:
it is the ZYX conversion (I originally made the same mistake):
![quaternion](https://user-images.githubusercontent.com/5664353/83559529-02e9e280-a515-11ea-8530-20b92b70f592.png)

Reply:
> it does not compare arrays of floats. An alternative is `numpy.testing`.
> (i.e. I do not like much split into 3 lines)

It seems wrong to import something "from above". It is my understanding that modules in `osgar.lib` package should not import stuff from `osgar`.

Reply:
I changed the code to multiple lines ... I also realized that this is the place where I have seen it (in older tests).